frontend/app/src/main/java/com/intersoft/groupup_app/MainActivity.kt

```diff
@@ -31,6 +32,16 @@ import com.intersoft.groupup_app.ui.theme.GroupUP_appTheme
 
 class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
+        var tempEventId = 0
+        var tempEventName = ""
+        var tempDescription = ""
+        var tempSelectedDateInMillis : Long = 0
+        var tempStartTimeInMillis : Long = 0
+        var tempDurationInMillis : Long = 0
+        var tempMaxNumberOfParticipants = 0
+        var tempLocation = ""
+        var tempHostId = 0
```

Comment:
It would probably be a lot cleaner to have all this data in a single EventModel variable.

Reply:
EventModel is in :data:event, this would break the planed system architecture